config/config.go

```diff
@@ -217,7 +216,7 @@ type ScrapeConfig struct {
 	// Indicator whether the scraped metrics should remain unmodified.
 	HonorLabels bool `yaml:"honor_labels,omitempty"`
 	// A set of query parameters with which the target is scraped.
-	Params url.Values `yaml:"params,omitempty"`
+	Params map[string]string `yaml:"params,omitempty"`
```

Comment:
The only use case we have right now for multiple params is federation (and we'll have roughly the same use case in future for restricting normal scraping). Considering a label value could be anything, is there a label separator that could work?